src/App.tsx

```diff
@@ -22,8 +28,16 @@ function App() {
 								className="border-t h-full"
 							>
 								<Panel defaultSize={50} minSize={25}>
+									<EsquerySelectorInput />
 									<Editor
 										value={code[language]}
+										highlightedRanges={
+											!astParseResult.ok
+												? undefined
+												: convertNodesToRanges(
+														astParseResult.esqueryMatchedNodes,
+													)
```

Comment:
can we switch the case? and remove the `!`?
```
astParseResult.ok ?  convertNodesToRanges(astParseResult.esqueryMatchedNodes) : undefined
```

Reply:
done